modules/microshift-manifests-override-paths.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-applications.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-manifests-override-paths_{context}"]
+= Override the list of manifest paths
+You can override the list of default paths with the following procedure.
+
+.Procedure
+
+. You can set `manifests.kustomizePaths` to `++"++/opt/alternate/path++"++` in the configuration file for a single path. Alternatively, you can set `kustomizePaths` to `++"++/opt/alternative/path.d/++*"++` in the configuration file for a glob pattern.
++
+[source,terminal]
+----
+manifests:
+    kustomizePaths:
+        - <location> <1>
+----
+<1> Set each location entry to an exact path using `++"++/opt/alternate/path++"++` or a glob pattern using `++"++/opt/alternative/path.d/++*"++`.
```

Comment:
This still reads as those these are actual values to use, but they're just examples of paths or patterns.

Reply:
See above some ideas.